packages/opentelemetry-exporter-stackdriver-trace/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/exporter-stackdriver-trace",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Comment:
I miss something https://github.com/open-telemetry/opentelemetry-js/pull/859, should it be removed ?

Reply:
The branch is simply out of date